udata/core/dataservices/models.py

```diff
@@ -181,6 +182,9 @@ class Dataservice(WithMetrics, Owned, db.Document):
         filterable={
             "key": "dataset",
         },
+        by_link=lambda dataservice: url_for(
```

Comment:
Should we name it `href` instead of `by_link`? Since it appears as a `href` subsection?

Reply:
Changed in 704d75bb5